anki_vector: #namespace

    n_actions: 3 # We have 3 actions, Forwards,TurnLeft,TurnRight
    n_observations: 1

    speed_step: 1.0 # Time to wait in the reset phases

    linear_forward_speed: 0.15 # Spwwed for ging fowards
    linear_turn_speed: 0.1 # Lienar speed when turning
    angular_speed: 2.5 # Angular speed when turning Left or Right
    init_linear_forward_speed: 0.0 # Initial linear speed in which we start each episode
    init_linear_turn_speed: 0.0 # Initial angular speed in shich we start each episode

    new_ranges: 1 # How many laser readings we jump in each observation reading, the bigger the less laser resolution
    min_range: 0.1 # Minimum meters below wich we consider we have crashed
    max_laser_value: 0.4 # Value considered Ok, no wall
    min_laser_value: 0 # Value considered there is an obstacle or crashed
    max_linear_aceleration: 12.0 # Linear acceleration value in which we consider ( acceleration vector contains gravity )

    forwards_reward: 5 # Points Given to go forwards
    turn_reward: 1 # Points Given to turn as action
    end_episode_points: 200 # Points given when ending an episode
